src/Day01.kt

```diff
@@ -2,23 +2,25 @@
 fun main() {
     fun part1(input: List<String>): Int {
         var checkSum = 0
-        for (values in input) {
-            val num = values.split(' ')
-            checkSum += num.maxOf { it.toInt() } - num.minOf { it.toInt() }
+        for (line in input) {
+            val values = line.split("\t")
+            checkSum += values.maxOf { it.toInt() } - values.minOf { it.toInt() }
         }
         return checkSum
     }
 
     fun part2(input: List<String>): Int {
         var checkSum = 0
-        for (values in input) checkSum += (values.split(' ').maxOf { it.toInt() } - values.split(' ')
-            .minOf { it.toInt() })
+        for (line in input) {
+            checkSum += (line.split("\t").maxOf { it.toInt() }
+                        - line.split("\t").minOf { it.toInt() })
+        }
```

Comment:
While I appreciate the effort to keep code concise this line was perhaps was too long to be readable, especially since it was automatically separated over multiple lines in the IDE due to length. 

When working with long lines you can put in arbitrary line breaks where it makes sense. In this case I broke on the minus sign. When working with multiple dot notation calls you can choose to break on every dot call.